resources/data/elephpants.json

```diff
@@ -148,14 +148,14 @@
             "id": 19,
 			"name": "MurPHPy",
 			"description": "AmsterdamPHP",
-			"sponsor": "AmsterdamPHP",
+			"sponsor": "AmsterdamPHP Community",
 			"year": 2014,
 			"image": "https://i.imgur.com/GdFxJmk.jpg"
 		},
 		{
             "id": 20,
 			"name": "Phil Snow",
-			"description": "ConFoo",
+			"description": "ConFoo web techno conference",
```

Comment:
please, fix caps here :-) or is it the right way to write?

Reply:
adjusted